red/api/flow.js

```diff
@@ -29,7 +29,11 @@ module.exports = {
         var flow = redNodes.getFlow(id);
         if (flow) {
             log.audit({event: "flow.get",id:id},req);
-            res.json(flow);
+            if (settings.flowFilePretty) {
+                flowDef = JSON.stringify(flowDef,null,4);
+            }
+            res.setHeader('Content-Type', 'application/json');
+            res.send(flow);
```

Comment:
In that case would you be opposed to adding in a new call to the library to return a stringified flows instead so that the api does not need to modify the object?